Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:2.7
+
+RUN pip install enos && \
```

Comment:
Enos should be installed from the corresponding git revision.
This could be something like :
```
COPY . /tmp/enos
WORKDIR /tmp/enos
RUN pip install .
```